orchestra/tests/test_project_api_client.py

```diff
@@ -85,26 +86,17 @@ def get(url, *args, **kwargs):
         self.assertEqual(len(res), 1)
         self.assertEqual(res[0]['id'], todo3.id)
 
-        # Filter by step slug
-        res = get_todos(project_id=None, step_slug=step_1.slug)
-        self.assertEqual(len(res), 3)
-        expected_ids = [todo1.id, todo2.id, todo4.id]
-        for r in res:
-            self.assertIn(r['id'], expected_ids)
-
         # Filter by project_id
         res = get_todos(project_id=project2.id)
         self.assertEqual(len(res), 2)
         expected_ids = [todo3.id, todo4.id]
         for r in res:
             self.assertIn(r['id'], expected_ids)
 
-        # Get all todos
-        res = get_todos()
-        self.assertEqual(len(res), 4)
-        expected_ids = [todo1.id, todo2.id, todo3.id, todo4.id]
-        for r in res:
-            self.assertIn(r['id'], expected_ids)
+        # Test project_id is required
```

Comment:
(Not a comment on the test, just a reminder that we mentioned we'd verify if we require project_id on the server side as well)